cpu/qn908x/Kconfig.clk

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020 iosabi
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "QN908x clock configuration"
+    depends on CPU_FAM_QN908X
+
+config BOARD_HAS_XTAL32K
+    bool "Board has an external 32.768 KHz crystal"
+    help
+         Indicates that the board has an external low frequency 32.786 KHz
+         crystal oscillator connected to the XTAL32_IN / XTAL32_OUT pins.
+
+choice
+    prompt "32K low frequency clock selector"
+    default CPU_CLK_32K_XTAL if BOARD_HAS_XTAL32K
+    default CPU_CLK_32K_RCO
+    help
+        The "32K" clock bus runs at either 32 KHz from the internal RCO or
+        32.768 KHz from an external crystal oscillator. This clock can be used
+        to drive the "System clock" for a very low power operation, but it can
+        independently also be used for the watchdog timer (WDT) and other low
+        frequency system timers like a real time clock.
+
+config CPU_CLK_32K_XTAL
+    bool "External 32.768 KHz crystal"
+    depends on BOARD_HAS_XTAL32K
+
+config CPU_CLK_32K_RCO
+    bool "Internal 32 KHz oscillator"
+
+endchoice
+
+config BOARD_HAS_XTAL
+    bool "Board has an external 32 MHz or 16 MHz crystal"
```

Comment:
Same here:
```suggestion
    bool
```

Reply:
Done.

Reply:
I changed the BOARD_HAS_XTAL to be implied by BOARD_HAS_XTAL_32M or BOARD_HAS_XTAL_16M and made all three have no prompt. The xtal is certainly soldered to the board so it is part of the board definition to tell which one is there.